### PR TITLE
Add location rating

### PR DESCRIPTION
**Pull request by TheTedder**
added average_rating method to location object as a stub.  
added a serializer for location.
updated tests to reflect changes in object/serializer

### CODE REVIEW

File: app/models/location.rb [hunk 1]
```diff
@@ -8,4 +8,9 @@ class Location < ApplicationRecord
   validates :price, numericality: { greater_than_or_equal_to: 0, only_integer: true }, allow_nil: true
   validates :password_protected, inclusion: { in: [true, false] }, allow_nil: true
   validates :zip, numericality: { only_integer: true }, presence: true
+
+  def average_rating
+    #stub
```
Comment (safang99) [inline app/models/location.rb]: remove stub methods until they are ready to implement

File: app/serializers/location_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class LocationSerializer < ActiveModel::Serializer
+  attributes :id, :name, :city, :state, :price, :password_protected, :average_rating
```
Comment (safang99) [inline app/serializers/location_serializer.rb]: remove average_rating

File: spec/controllers/api/v1/locations_controller_spec.rb [hunk 3]
```diff
@@ -35,10 +35,11 @@
...
 
-      expect(returned_json.length).to eq(2)
-      expect(returned_json[0]['name']).to eq('Nero')
-      expect(returned_json[1]['name']).to eq('Cafe')
-      expect(returned_json[1]['password_protected']).to eq(true)
+      expect(returned_json['locations'].length).to eq(2)
+      expect(returned_json['locations'][0]['name']).to eq('Nero')
+      expect(returned_json['locations'][1]['name']).to eq('Cafe')
+      expect(returned_json['locations'][1]['password_protected']).to eq(true)
+      expect(returned_json['locations']).to all(include('average_rating'))
```
Comment (safang99) [inline spec/controllers/api/v1/locations_controller_spec.rb]: remover average_rating test

File: app/serializers/location_serializer.rb (hunk 2 above)
Comment (nwalberts) [inline app/serializers/location_serializer.rb]: uhhhhh is there a reason that `password_protected` is here? Otherwise pls remove

Comment (safang99) [reply app/serializers/location_serializer.rb]: We are passing a "password_protected" status down meaning if a wifi requires a password or not

**Pull request merged**